### PR TITLE
Add Skilljar detector

### PR DESCRIPTION
**Pull request by chuglo**
### Description:
This PR adds a detector for [Skilljar](https://www.skilljar.com/). With a Skilljar API key, it can be possible to do things such as exfiltrate PII from the users list.

The verification uses a /ping endpoint designed to test validity of keys. 
https://support.skilljar.com/hc/en-us/articles/203811260-Getting-started-with-the-Skilljar-API

### Checklist:
* [X] Tests passing (`make test-community`)?
* [X] Lint passing (`make lint` this requires [golangci-lint](https://golangci-lint.run/usage/install/#local-installation))?

### CODE REVIEW

**Comment by dustin-decker** [comment]
Thank you for your contribution @chuglo. We'll add a test key and review this soon.

File: pkg/detectors/skilljar/skilljar_test.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+func TestSkilljar_FromChunk(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors4")
```
Comment (zricethezav) [inline pkg/detectors/skilljar/skilljar_test.go:23]: @chuglo would you mind changing this to `detectors5`? 
```suggestion
	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors5")
```

**Comment by zricethezav** [comment]
@chuglo Thanks for the PR! Would you mind resolving the conflicts as we recently merged some new detectors?

**Comment by chuglo** [comment]
I definitely misclicked. Let me see if I can fix the PR closure..

**Pull request closed without merging**